lockc/Cargo.toml

```diff
@@ -22,25 +22,32 @@ bindgen = "0.59"
 byteorder = "1.4"
 chrono = { version = "0.4", default-features = false, features = ["clock"] }
 config = { version = "0.11", default-features = false, features = ["toml"] }
+ctrlc = "3.2"
 dirs = "4.0"
+fanotify-rs = { git = "https://github.com/vadorovsky/fanotify-rs", branch = "fix-pid-type" }
 futures = "0.3"
+goblin = "0.4"
 kube = "0.63"
 k8s-openapi = { version = "0.13", default-features = false, features = ["v1_21"] }
 lazy_static = "1.4"
 libc = { version = "0.2", features = [ "extra_traits" ] }
 libbpf-rs = "0.14"
+lockc-uprobes = { path = "../lockc-uprobes" }
```

Comment:
Hmm... the doc you linked is, from my perspective, doing what I do - it defines a workspace on top of the repo and then keeps crates in subdirectories. Exactly how we are doing here with the main `Cargo.toml` referencing `lockc`, `lockc-uprobe` and `xtask` crates. I don't see any `./crates/` directory in that doc.

The layout is also similar to what aya template does:

https://github.com/aya-rs/aya-template
https://github.com/willfindlay/suidsnoop
https://github.com/willfindlay/aya-lsm-args-example

I was thinking about having a workspace with 5 crates when migrating to aya:
* `lockc`
* `lockc-common` (for structs shared between BPF and userspace)
* `lockc-ebpf`
* `lockc-uprobes` (this one has to be separate, otherwise uprobes don't work :/)
* `xtask`

With the same layout - workspace `Cargo.toml` in the top directory, crates in subdirectories.

Reply:
Oh sorry, I didn't get it